script.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * An octoherd script to watch, unwatch or ignore a repository
+ *
+ * @param {import('@octoherd/cli').Octokit} octokit
+ * @param {import('@octoherd/cli').Repository} repository
+ * @param {{unwatch?: boolean, ignore?: boolean}} options Custom user options passed to the CLI
+ */
+export async function script(octokit, repository, { unwatch, ignore }) {
+	if (unwatch && ignore) {
+		octokit.log.error("Not possible tu unsubscribe and subscribe ignoring notifications")
+		return;
```

Comment:
You want to throw an error here so that the script stops, otherwise you will see the same error for every repository.

```suggestion
		throw new Error("Use either --unwatch or --ignore, not both")
```